### PR TITLE
erratum: stash json to enable get_erratum_data()

### PR DESCRIPTION
**Pull request by sosiouxme**

### CODE REVIEW

File: errata_tool/erratum.py [hunk 1]
```diff
@@ -211,6 +212,7 @@ def _fetch(self, errata_id):
             # NEW_FILES QE etc.
             self.errata_state = erratum['status']
             self._original_state = self.errata_state
+            self._original_json = erratum
```
Comment (lhh) [inline errata_tool/erratum.py]: This is only capturing the API/v1 content. The old style errata content is not here. It's entirely fine if that's what you need, but I just wanted to make a note of that.

Comment (lhh) [reply errata_tool/erratum.py]: There are a few fields that are not in the API/v1 content; see line 270-ish.

**Comment by yazug** [comment]
this partially addresses issue #156

**Comment by ktdreyer** [comment]
I've added docs and tests in https://github.com/red-hat-storage/errata-tool/pull/160

**Pull request merged**